usaspending_api/awards/v2/views/subawards.py

```diff
@@ -66,10 +66,15 @@ def _business_logic(self, request_data):
 
         queryset = queryset.values(*list(self.subaward_lookup.values()))
 
+        # always secondary-sort by PK in case a repeating value (e.g. subaward_number) crosses pages, so suborder isn't abitrary
         if request_data["order"] == "desc":
-            queryset = queryset.order_by(F(self.subaward_lookup[request_data["sort"]]).desc(nulls_last=True))
+            queryset = queryset.order_by(
+                F(self.subaward_lookup[request_data["sort"]]).desc(nulls_last=True), "subaward_id"
```

Comment:
The `subaward_id` sort should mimic the primary sort field direction too.

Otherwise a flipping the sort direction isn't inverting the results.

Reply:
Since they weren't sorting by anything else, I didn't bother before. But I changed it. @tony-sappe

Reply:
Right, the secondary sort is being added. Still needs to  intuitive behavior. Thanks